### PR TITLE
Added Starboard

### PR DESCRIPTION
**Pull request by xsnowstorm**
**What kind of change does this PR introduce?**  
Feature

**Describe the changes proposed in the pull request**  
If a message reaches 5 star reactions it will get posted to the starboard channel.

**What changes need to be made to the application**
Add value for starboard channel env field.

**Does this PR introduce changes to the database?**
yes

**Contributers**
xsnowstorm
demirdegerli

### CODE REVIEW

**Comment by demirdegerli** [comment]
good work 😎

**Comment by demirdegerli** [comment]
all message scenarios are tested (no content, no attachment, only attachment, video, image...)

**Comment by xsnowstorm** [comment]
> all message scenarios are tested (no content, no attachment, only attachment, video, image...)

Great :)

**Comment by demirdegerli** [comment]
i forgot to commit the latest code, it's now ready

File: bot/__main__.py [hunk 1]
```diff
@@ -54,6 +54,12 @@
             "membercount" INT
     )
     """,
+    """
+    CREATE TABLE IF NOT EXISTS "starboard" (
+        "message_id" INT UNIQUE,
```
Comment (logicguy1) [inline bot/__main__.py:59]: This table has no primarry key

**Review by logicguy1 — COMMENTED** [review]
yeeaah slight database thingy

**Comment by demirdegerli** [comment]
@logicguy1 close this pr

**Pull request closed without merging**